### PR TITLE
Add tests for line break inside series command

### PR DESCRIPTION
**Pull request by vtols**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/series/command/LineBreakInsideSeriesCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        List<PlainCommand> commandList = new ArrayList<>();
+        commandList.add(firstCommand);
+        commandList.add(ordinarySeriesCommand);
+        CommandSendingResult res = send(commandList);
+
+        assertEquals(0, (int) res.getFail());
+        assertEquals(2, (int) res.getSuccess());
+        assertEquals(2, (int) res.getTotal());
+    }
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/series/command/LineBreakInsideSeriesCommandTest.java]: Please, add assert messages to be able understand error if occurs.

**Comment by vtols** [comment]
Should I split Network and Data API testing or leave as it is?

**Comment by vtols** [comment]
Failing test are uncommented now, the issue https://nur.axibase.com:41791/redmine/issues/3906 should be resolved

**Comment by rmakulov** [comment]
Please, fix compilation errors.

**Pull request merged**